<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit_23">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>100</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: outset;
border-width: 1px;
border-radius: 10px;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
    <property name="placeholderText">
     <string>Mot de pass</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>100</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Mot de pass            *</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_26">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>220</y>
      <width>146</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(165, 120, 255);
border-style: outset;
border-width: 2px;
border-radius: 10px;
border-color: beige;
min-width: 10em;
padding: 6px;
}
QPushButton:hover:!pressed
{
background-color: rgb(170, 170, 255);
}</string>
    </property>
    <property name="text">
     <string>Réinitialiser </string>
    </property>
    <property name="icon">
     <iconset resource="iconne.qrc">
      <normaloff>:/icone/reset.png</normaloff>:/icone/reset.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>22</width>
      <height>22</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_27">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>391</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(112, 255, 133);
border-style: outset;
border-width: 2px;
border-radius: 10px;
border-color: beige;
min-width: 10em;
padding: 6px;

}
QPushButton:hover:!pressed
{

	
	background-color: rgb(55, 217, 109);
}</string>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
    <property name="icon">
     <iconset resource="iconne.qrc">
      <normaloff>:/icone/login.png</normaloff>:/icone/login.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nom Utilisateur       *</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_22">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: outset;
border-width: 1px;
border-radius: 10px;</string>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
    <property name="placeholderText">
     <string>Utilisateur</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>220</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 170, 255);</string>
    </property>
    <property name="text">
     <string>Mot de pass oublié ? </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_25">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>181</y>
      <width>381</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="iconne.qrc"/>
 </resources>
 <connections/>
</ui>
